src/main/java/com/sphy/hotelmanagementapplication/dto/HotelDTO.java

```diff
@@ -21,7 +23,7 @@ public class HotelDTO implements Serializable {
     private Set<RoomDTO> rooms =new HashSet<>();
 
     public HotelDTO(long id) {
-
+        super.setId(id);
```

Comment:
This is not right, a dto should not extend BaseEntity, we might want to add more functionality to it, and that might break things